script/Arcueid/Day 1/01_00_ARC01_2_2.txt

```diff
@@ -217,7 +217,7 @@ To be honest, I'm surprised she even remembers my name.
 [sha:098bd92b93fd1eff159a437142ad256ea648d771]{
 -- Page 176, Offset 1624.
 -- いま引っ越しって言ったけど、遠野くん、どこかに引っ越しちゃうの？　もしかして、転校とか？」
-You just said you're moving, but are you going somewhere else, Tohno-kun? Are you changing schools?"
+You just said you're moving, but are you going somewhere else, Tohno-kun? Are you transferring schools?"
```

Comment:
"transferring schools" is awkward but "changing schools" is quite natural, so you should revert. It could also be just "Are you transferring?" or "Are you transferring to another school?"